bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerMetadata.java

```diff
@@ -691,4 +690,23 @@ Long getLastEnsembleKey() {
         checkState(!ensembles.isEmpty(), "Metadata should never be created with no ensembles");
         return ensembles.lastKey();
     }
+
+    int getMetadataFormatVersion() {
+        return metadataFormatVersion;
+    }
+
+    private static DigestType protoToApiDigestType(LedgerMetadataFormat.DigestType digestType) {
```

Comment:
I would suggest pulling this method into a util class. so it can be used across the project.

Reply:
Not sure where it would fit, so leaving it here for now. We have too many of these conversions though. We should push them all into a DigestConvertor class at some point, but I don't want to do that in this patch as it would mean touching other classes which are completely unrelated.